src/policy.rs

```diff
@@ -136,8 +145,8 @@ impl Policy {
         Ok(())
     }
 
-    /// Rotate an attribute, changing its underlying value with that of an
-    /// unused slot
+    /// Rotates an attribute, changing its underlying value with that of an
+    /// unused slot.
```

Comment:
This comment is confusing, I don't understand the: `with that of an unused slot`.

`marking its underlying value as deprecated` is what it means?

Reply:
No, it means the new value will be a value not already taken by any attribute. But I agree, this is not clear, I will change.